.github/workflows/build.yml

```diff
@@ -22,6 +22,30 @@ jobs:
       working-directory: build/check_install_root
       run: diff -u ../../test/objc_list.txt <(du -a | tail -r | awk -F ' ' '{print $2}')
 
+  build-on-osx-for-python:
+    runs-on: macos-10.15
+    strategy:
+      matrix:
+        python-version: [3.6]
```

Comment:
That's quite a small matrix. Do you plan on extending this?

Reply:
I was aiming at an MVP for this PR, but we could add the other major versions: 3.7, 3.8 and 3.9.